shell/platform/android/io/flutter/embedding/android/KeyEmbedderResponder.java

```diff
@@ -151,8 +152,8 @@ void synchronizePressingKey(
     // 2. Derive the pre-event state of the event key (if applicable.)
     for (int keyIdx = 0; keyIdx < goal.keys.length; keyIdx += 1) {
       final KeyboardMap.KeyPair key = goal.keys[keyIdx];
-      nowStates[keyIdx] = pressingRecords.containsKey(goal.keys[keyIdx].physicalKey);
-      if (goal.keys[keyIdx].logicalKey == eventLogicalKey) {
+      nowStates[keyIdx] = pressingRecords.containsKey(key.physicalKey);
+      if (key.logicalKey == eventLogicalKey) {
```

Comment:
Just to confirm, are these two lines of change make any difference, or purely cosmetic? (I'm ok with either)

Reply:
Purely cosmetic. The 'key' variable was already defined (before this PR) and It could be used on this two lines.